app/src/main/java/com/axiel7/anihyou/ui/screens/mediadetails/edit/EditMediaViewModel.kt

```diff
@@ -89,7 +89,7 @@ class EditMediaViewModel @Inject constructor(
                     status = value,
                     progress = uiState.mediaHasDuration() ?: uiState.progress,
                     volumeProgress = uiState.mediaHasVolumes() ?: uiState.volumeProgress,
-                    completedAt = LocalDate.now()
+                    completedAt = uiState.completedAt ?: LocalDate.now()
```

Comment:
This fix will affect other status than "Repeating", but this reflects anilist website behaviour